artifact/v3/manifest/manifest.cpp

```diff
@@ -80,6 +96,14 @@ ExpectedManifest Parse(mender::common::io::Reader &reader) {
 	return m;
 }
 
+string Manifest::Get(const string &key) {
+	auto value = this->map_.find(key);
+	if (value != this->map_.end()) {
```

Comment:
I'd do `if (this->map_.count(key) != 0)`

Reply:
This is pre-existing. I've only moved it into the `cpp` file.

Also, it's how the `cppreference examples` do it